packages/tools/src/utilities/contourSegmentation/copyAnnotation.ts

```diff
@@ -0,0 +1,149 @@
+import { utilities } from '@cornerstonejs/core';
+import type {
+  ContourSegmentationAnnotation,
+  ContourSegmentationData,
+} from '../../types';
+import { getSegmentation } from '../../stateManagement/segmentation/getSegmentation';
+import { addAnnotation, getAnnotation } from '../../stateManagement';
+import { getViewportAssociatedToSegmentation } from './getViewportAssociatedToSegmentation';
+import { getToolGroupForViewport } from '../../store/ToolGroupManager';
+
+export function copyAnnotation(
```

Comment:
so it is not copy ANY annotation right? since i see it is specific to polyline, can we rename this

Reply:
Yes it is. Its returning a new annotation object based on the given annotation